tests/TestUtil.php

```diff
@@ -45,7 +45,7 @@ public static function getConnection(): PDO
             $dsn .= 'dbname=' . $connectionParams['dbname'] . $separator;
             $dsn .= self::getCharsetValue($connectionParams['charset'], $connectionParams['driver']) . $separator;
             $dsn = rtrim($dsn);
-            self::$connection = new PDO($dsn, $connectionParams['user'], $connectionParams['password']);
+            self::$connection = new PDO($dsn, $connectionParams['user'], $connectionParams['password'], [PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION]);
```

Comment:
@prolic should the error mode be an env variable ? should we have by default silent mode but add another entry in the matrix for exception mode ?